Slim/Formats.pm

```diff
@@ -79,7 +79,7 @@ sub init {
 		'xpf' => 'Slim::Formats::Playlists::XSPF',
 	);
 
-	if ($Audio::Scan::VERSION =~ /^0\.9[45]$/) {
+	if ($Audio::Scan::VERSION >= 0.94) {
```

Comment:
We should probably use `Slim::Utils::Versions` instead, as that would cover a potential version string of `1.9.2`, too.